### ISSUE TITLE
CRAN checklist

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/datawizard#16]
I want to push this to `CRAN` asap. So listing things that need to be done before I push it to `CRAN`. 
Please edit as you please:

- [x] improve `README`
- [x] `devtools::check_win_devel()`
- [x] `devtools::check_win_release()`

**Comment by IndrajeetPatil** [comment]
@DominiqueMakowski Anything else? I think we are ready for submission.

I don't think we need a vignette, since the README and the examples are sufficient.

**Comment by DominiqueMakowski** [comment]
Move the function definition:

- [ ] https://github.com/easystats/modelbased/issues/116

**Comment by IndrajeetPatil** [comment]
Can you please move it? I will take care of the rest (updating docs, README, pkgdown, etc.).

**Comment by DominiqueMakowski** [comment]
done but it requires color printing and check packages from insight (?)

**Comment by IndrajeetPatil** [comment]
Yeah, no problem. `datawizard` already relies on `insight`.

**Comment by DominiqueMakowski** [comment]
ah right

**Comment by IndrajeetPatil** [comment]
@strengejacke Do you have anything else to add to `datawizard`? 
I will be submitting today if there is nothing else you'd like to add.

**Comment by strengejacke** [comment]
no, not now.

**Comment by IndrajeetPatil** [comment]
@DominiqueMakowski I now see the following `WARNING` in `R CMD check`:

```r
* checking dependencies in R code ... WARNING
Missing or unexported object: 'see::geoms_from_list'
```

Not sure how to get rid of this. We already have `see` in `Suggests`:
https://github.com/easystats/datawizard/blob/36a1725cbf74bd3d752633f26161b9ae4dbfd7d3/DESCRIPTION#L59

Also, is it just me, or a visualization function seems a bit out of place in a data wrangling/transformation package? 
Why are we not including this function in `see`?

**Comment by DominiqueMakowski** [comment]
lool It's not a function it's just a definition - which I'll need in other packages. But anyway yeah it requires a non cran see, so you can delete it for now

**Comment by IndrajeetPatil** [comment]
Okay, I am moving to WIP for now and will add it back once it's on CRAN. Then you can delete it from `modelbased`.
